lib/CMakeLists.txt

```diff
@@ -25,6 +25,13 @@ else()
 				set(ispcVersion ${curVersion})
 			endif()
 		endforeach()
+		list(APPEND ispcInstallPath ${ISPC_TEXCOMP_DIR}/ISPC/win)
+	endif()
+	if(CMAKE_SYSTEM MATCHES "Linux")
```

Comment:
Would be cleaner to use elseif() rather than else() followed by if().

Reply:
yes it would :)